ale_linters/python/mypy.vim

```diff
@@ -6,7 +6,7 @@ let g:ale_python_mypy_args = get(g:, 'ale_python_mypy_options', '')
 function! g:ale_linters#python#mypy#GetCommand(buffer) abort
     return g:ale#util#stdin_wrapper
     \   . ' .py mypy --show-column-numbers '
-    \   . g:ale_python_mypy_options
+    \   . g:ale_python_mypy_args
```

Comment:
That's the wrong fix there. Change the `let` statement above. See this: https://github.com/w0rp/ale/issues/257